draft-ietf-quic-http.md

```diff
@@ -583,12 +673,34 @@ server to client, as in HTTP/2.  It defines no flags.
 
 The payload consists of:
 
-  Promised Stream ID:
-  : A 32-bit Stream ID indicating the QUIC stream on which the response will be
-    sent
+Push ID:
+: A 32-bit identifier for the server push request.  A push ID is used in push
```

Comment:
It is a request -- the PUSH_PROMISE contains a request body that the server is telling the client it expects the client will want to make shortly.